rest/api_test.go

```diff
@@ -117,6 +117,8 @@ func TestDocLifecycle(t *testing.T) {
 
 //Validate that Etag header value is surrounded with double quotes, see issue #808
 func TestDocEtag(t *testing.T) {
+	defer base.SetUpTestLogging(base.LevelDebug, base.KeyAccess)()
```

Comment:
`KeyAccess` is used to log information about user access grant changes (i.e. new channel access, role changes, etc.) - given we're not sure what causes this flaky test, I'd suggest `KeyAll` to enable all types of logging.

```suggestion
	defer base.SetUpTestLogging(base.LevelDebug, base.KeyAll)()
```